commands.js

```diff
@@ -551,39 +551,28 @@ var commands = exports.commands = {
 		var targetId = toId(target) || user.userid;
 		var targetUser = Users.getExact(targetId);
 		var targetUsername = (targetUser ? targetUser.name : target);
+		var canPrivate = targetId === user.userid || user.can('makeroom');
 
 		var buffer = [];
-		var innerBuffer = [];
+		var publicAuth = [];
+		var privateAuth = [];
 		var group = Users.usergroups[targetId];
-		if (group) {
-			buffer.push('Global auth: ' + group.charAt(0));
-		}
-		for (var i = 0; i < Rooms.global.chatRooms.length; i++) {
+		if (group) buffer.push('Global auth: ' + group.charAt(0));
+
+		for (var i = 0, len = Rooms.global.chatRooms.length; i < len; i++) {
```

Comment:
How so? I thought caching ``Rooms.global.chatRooms.length`` in ``len`` was better for performance than checking it on each iteration of the loop. If it's not, what would be wrong with looping backwards (``for (var i = Rooms.global.chatRooms.length; i--;)``) besides the order of the rooms?